tests/unit/test_common.c

```diff
@@ -43,6 +43,38 @@
 /** The value of the global attribute in the sample 2 output file. */
 #define ATT_VALUE_S2 42
 
+/* How many flavors of netCDF are available? */
+int
+get_iotypes(int *num_flavors, int *flavors)
+{
+    int num = 0;
+    int fmtidx = 0;
+    int format[NUM_FLAVORS];
+
+#ifdef _PNETCDF
+    num++;
+    format[fmtidx++] = PIO_IOTYPE_PNETCDF;
+#endif
+#ifdef _NETCDF
+    num++;
+    format[fmtidx++] = PIO_IOTYPE_NETCDF;
+#endif
+#ifdef _NETCDF4
```

Comment:
Hmmm, the fortran testing framework assumes PIO_IOTYPE_NETCDF when _NETCDF4 is enabled (I always assumed if you have _NETCDF4 it means you also have _NETCDF. Is there a case where _NETCDF4 is enabled by not _NETCDF? Or does _NETCDF4 imply that _NETCDF is also enabled?). I will fix that.